packages/swatch/src/Swatch.ts

```diff
@@ -223,12 +223,19 @@ export class Swatch extends SizedMixin(Focusable, {
             );
         }
         if (changes.has('label')) {
-            if (this.label) {
+            if (this.color !== '') {
```

Comment:
I _think_ you placed this logic in the wrong order and it's what's causing the test failures in CI right now. Can you take a look at those?

Reply:
Should be good now except for the fact that few NumberFields tests are failing. Switching the order of if else did the trick!
Thanks